### PR TITLE
Add let's encrypt instructions to Enterprise platform tips

### PR DESCRIPTION
**Pull request by schultyy**
This PR adds instructions on how to obtain and configure a Let's Encrypt certificate for a Travis CI Enterprise installation in the platform tips section.

### CODE REVIEW

File: user/enterprise/platform-tips.md [hunk 1]
```diff
@@ -254,3 +254,15 @@ After that, scroll down and click "Save". After your changes have been saved, yo
...
+
+**Note: This will also cause downtime**
+
+```
+$ replicatedctl app stop
+$ sudo certbot renew
+$ replicatedctl app start
+```
+
+In general: These certificate renewals should be automated with a cron job.
```
Comment (bnferguson) [inline user/enterprise/platform-tips.md]: You ended up not going with the pre and post hooks?

Comment (schultyy) [reply user/enterprise/platform-tips.md]: No, I decided to do it like this for now which makes it a bit more obvious what's going on. I didn't feel a 100% comfortable about the pre and post hooks yet.

Comment (bnferguson) [reply user/enterprise/platform-tips.md]: 👍 That makes sense.

**Comment by danishkhan** [comment]
👍 went through it and it worked great. I did notice that it looks like it only works for root domain or at least when I set it up [Video link](https://cl.ly/0o3T0Z1Y2H1l)

**Comment by schultyy** [comment]
@danishkhan Hmm that's strange. I've checked against our production instance and there I can't see that behavior.

**Comment by schultyy** [comment]
@mariana-gu Fantastic! Thanks so much!

**Pull request merged**